watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/SepRO_plus_Chlorination.py

```diff
@@ -70,7 +70,7 @@
 from pyomo.network import Arc
 
 from watertap.examples.flowsheets.full_treatment_train.util import (
-    solve_with_user_scaling,
```

Comment:
This file (SepRO_plus_Chlorination.py) is removed by PR #284 (redundant and unused in full flowsheet)